cron/photos.py

```diff
@@ -54,6 +55,33 @@ def download_photo(user):
         # Some students don't have photos if they never went for picture day
     return None
 
+    def download_photo(url):
+        try:
+            return download_photo_bytes(url)
+        except PIL.UnidentifiedImageError:
+            print("Unable to download {} with primary url")
+        try:
+            return download_photo_bytes(url)
+        except PIL.UnidentifiedImageError:
+            print("UNABLE to download " )
+            # If there is another issue 
+        return None
+
+    
+
+    def read_csv_as_dict():
+        photoids = {}
+        if 'idphotos_2021.csv':

```

Comment:
this will always be true

Reply:
Should i say != null? I want to tell it to use it if it can find it...

Reply:
we'll do this a different way that allows for different filenames, just remove this for now.